package.json

```diff
@@ -2722,7 +2722,7 @@
     "watch-docs": "mkdocs serve",
     "clean": "rimraf ./out && rimraf ./tsconfig.tsbuildinfo && rimraf ./cljs-out",
     "update-grammar": "node ./src/calva-fmt/update-grammar.js ./src/calva-fmt/atom-language-clojure/grammars/clojure.cson clojure.tmLanguage.json",
-    "precompile": "npm i && npm run clean && npm run update-grammar && npm run prettier-format",
+    "precompile": "npm i && npm run clean && npm run update-grammar",
```

Comment:
Why remove this @PEZ ?

Reply:
It's not needed to compile. It's needed before we run the `prettier-format-watch` script. And there's a `preprettier-format-watch` for that.